sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -492,8 +492,19 @@ export class LocalWorkspace implements Workspace {
      * @param version the version of the plugin e.g. "v1.0.0".
      * @param kind the kind of plugin, defaults to "resource"
      */
-    async installPlugin(name: string, version: string, kind = "resource"): Promise<void> {
-        await this.runPulumiCmd(["plugin", "install", kind, name, version]);
+    async installPlugin(name: string, version: string): Promise<void> {
+        await this.runPulumiCmd(["plugin", "install", "resource", name, version]);
+    }
+    /**
+     * Installs a plugin in the Workspace, from a third party server.
+     *
+     * @param name the name of the plugin.
+     * @param version the version of the plugin e.g. "v1.0.0".
+     * @param kind the kind of plugin, defaults to "resource"
+     * @param server the server to install the plugin from
+     */
+    async installPluginFromServer(name: string, version: string, server: string): Promise<void> {
```

Comment:
In Go, you’re adding the new method to the `Workspace` interface, and then implementing it in `LocalWorkspace`, but here you're just adding it to `LocalWorkspace`. Should we add the new method to the `Workspace` interface in `workspace.ts` too?

Reply:
good catch! I belive this is resolved now